backend/tests/unit/exchangeRates.test.js

```
@@ -0,0 +1,366 @@
+          rate: 0.055000, // Different rate
+          timestamp: new Date().toISOString()
+        }
+      });
+
+      // Simulate time passing (5 minutes + 1 second)
+      const originalDate = Date;
+      const futureTime = new Date(Date.now() + 5 * 60 * 1000 + 1000);
+      global.Date = class extends Date {
+        constructor() {
+          return new originalDate(futureTime);
+        }
```

Biome (2.1.2)

[error] This constructor has code paths that return without calling `super()` first.

This statement returns from the constructor before `super()` has been called:

If this is intentional, add an explicit throw statement in unsupported paths.

(lint/correctness/noUnreachableSuper)